src/aws_scanner_config.py

```diff
@@ -136,14 +137,17 @@ def iam_password_policy_hard_expiry(self) -> bool:
     def kms_role(self) -> str:
         return self._get_config("kms", "role")
 
-    def logs_vpc_log_group_name(self) -> str:
-        return self._get_config("logs", "vpc_log_group_name")
+    def logs_group_name(self, service_name: ServiceName) -> str:
```

Comment:
- Perhaps better to error than return empty string here :shrug: 
- Perhaps this kind of business logic should be in the class rather than in config

Reply:
Invalid name handled by raising an exception instead of returning an empty string